src/components/calcite-action-menu/calcite-action-menu.tsx

```diff
@@ -95,6 +95,7 @@ export class CalciteActionMenu {
 
   /**
    * Determines where the component will be positioned relative to the referenceElement.
+   * @see [PopperPlacement](https://github.com/Esri/calcite-components/blob/master/src/utils/popper.ts#L25)
```

Comment:
Maybe these should just be relative paths like `../../utils/popper.ts#L25`? Then the consuming app (API pages) can just add the full url and version? Using `https://github.com/Esri/calcite-components/tree/` + <package.json.version> + `/...`?